PS C:\Users\xyf20\Desktop\RustDataProcess\pyo3_learn> cargo run
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.03s
     Running `target\debug\pyo3_learn.exe`
Python version: 3.12.6 (tags/v3.12.6:a4a2d2b, Sep  6 2024, 20:11:23) [MSC v.1940 64 bit (AMD64)]
Python executable: C:\Users\xyf20\Desktop\RustDataProcess\pyo3_learn\target\debug\pyo3_learn.exe
PyO3 Python path:
  C:\Users\xyf20\AppData\Local\Programs\Python\Python312\python312.zip
  C:\Users\xyf20\AppData\Local\Programs\Python\Python312\DLLs
  C:\Users\xyf20\AppData\Local\Programs\Python\Python312\Lib
  C:\Users\xyf20\Desktop\RustDataProcess\pyo3_learn\target\debug
  C:\Users\xyf20\AppData\Local\Programs\Python\Python312
  C:\Users\xyf20\AppData\Local\Programs\Python\Python312\Lib\site-packages
poe-api-wrapper version: 1.6.9
PyO3 Python path:
  C:\Users\xyf20\AppData\Local\Programs\Python\Python312\python312.zip
  C:\Users\xyf20\AppData\Local\Programs\Python\Python312\DLLs
  C:\Users\xyf20\AppData\Local\Programs\Python\Python312\Lib
  C:\Users\xyf20\Desktop\RustDataProcess\pyo3_learn\target\debug
  C:\Users\xyf20\AppData\Local\Programs\Python\Python312
  C:\Users\xyf20\AppData\Local\Programs\Python\Python312\Lib\site-packages
  C:\Users\xyf20\Desktop\RustDataProcess\python_virtual_env\Lib\site-packages
Successfully imported poe-api-wrapper after path addition
Starting to initialize PoeClient...
Attempting to import poe_api_wrapper...
Successfully imported poe_api_wrapper
2024-10-29 21:17:15.506 | INFO     | poe_api_wrapper.bundles:init_window:21 - Initializing web data
2024-10-29 21:17:21.103 | INFO     | poe_api_wrapper.bundles:init_window:41 - Web data initialized
2024-10-29 21:17:21.106 | INFO     | poe_api_wrapper.bundles:get_form_key:82 - Retrieved formkey successfully: 1db41e541f276f1b0519e478c76a3c25        

Iterating through settings dictionary:
Key: 'subscription', Value: {'isActive': True, 'planType': 'monthly', 'id': 'U3Vic2NyaXB0aW9uOjA=', 'purchaseType': 'web', 'isComplimentary': False, 'expiresTime': 1731389824000000, 'willCancelAtPeriodEnd': True, 'isFreeTrial': False, 'purchaseRevocationReason': None}
Key: 'messagePointInfo', Value: {'messagePointResetTime': 1731389950000000, 'messagePointBalance': 306806, 'totalMessagePointAllotment': 1000000, 'id': 'TWVzc2FnZVBvaW50SW5mbzow'}
Sending message to gpt4_o_mini: 你好，请介绍一下你自己
2024-10-29 21:17:25.641 | INFO     | poe_api_wrapper.api:send_message:739 - New Thread created | 3rbmhy87mznlilxw5id
Bot response: 你好！我是GPT-4o mini，一个大型语言模型，由OpenAI训练。我可以帮助回答问题、提供信息、撰写文本和进行对话。如果你有任何具体的问题或需要帮助的地方，请随时告诉我！